liba/include/bridge/alloca.h

```diff
@@ -0,0 +1 @@
+#include <malloc.h>
```

Comment:
This should be guarded on windows (`#ifdef _WIN32` or something) and you should fall back to `#include_next <alloca.h>` otherwise.

Reply:
Indeed, excellent suggestion @jacobly0 . Thank you!